### DISCUSSION TITLE
如何提高缓存系统的内存利用率和可扩展性 · NSDI '21 · https://draveness.me/papers-segcache/

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#648] [Announcements]
https://draveness.me/papers-segcache/

**Comment by zcong1993** [comment]
终于恢复更新了😁

**Reply by jkrllezvous** [reply]
[说说看, Redis多线程](https://post.techwavelabs.org/redis/multi-thread-redis/)

**Comment by draveness** [comment]
> 终于恢复更新了😁

😂😂😂

**Comment by tomatopunk** [comment]
帮左大贴下代码:
1. https://github.com/twitter/pelikan
2. https://github.com/thesys-lab/segcache

**Comment by LiuWei-Luke** [comment]
昨天还在想大佬是不是回老家结婚不更新了来着:smile:

**Comment by DDusky** [comment]
逐渐看不懂o(╥﹏╥)o

**Comment by dreamChenp12** [comment]
常见的缓存系统为每个对象都会带来异常显著的额外开销 -- 有语病。应该是为每个对象保持元数据？

---


*2021-07-19 UPDATES: 已修复*

**Comment by GrayXu** [comment]
这个工作里具体到怎么用batching来做还是挺有启发的。他们组好像把Segcache做到PM上了，有些特点比较match。
> refer to https://www.youtube.com/watch?v=lTiw4ehHAP4

btw, 文章尾部的reference错了

**Comment by jkrllezvous** [comment]
[可以说下Redis多线程吗](https://post.techwavelabs.org/redis/multi-thread-redis/)
